### PR TITLE
PER-8458-profile-completion

### PR DESCRIPTION
**Pull request by crisnicandrei**
I have removed some of the values from the profile completion checklist, because some of them were not used in the profile, preventing it to reach 100%. I have also made it easier for the profile to reach 100%, needing only 1 entry for each category.

Steps to test: 
1. Go to the Archive profile tab
2. Add one entry to each section
3. The profile percentage should then reach 100%

### CODE REVIEW

File: src/app/shared/services/profile/profile.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      .dontMock(ProfileService);
+  });
+
+  it('should exist', async () => {
+    const { instance } = await shallow.createService();
+
+    expect(instance).toBeTruthy();
+  });
+
+  it('should return the correct completion value', async () => {
```
Comment (meisekimiu) [inline src/app/shared/services/profile/profile.service.spec.ts:79]: Please give each test a unique name.

**Review by meisekimiu — CHANGES_REQUESTED** [review]
Left some feedback on the tests.

Have you tested this on different archive types? (Personal vs Group vs Organization). These different types have differing metadata fields.

**Comment by crisnicandrei** [comment]
@meisekimiu I did test on different archive types

File: src/app/shared/services/profile/profile.service.spec.ts (hunk 1 above)
Comment (crisnicandrei) [reply src/app/shared/services/profile/profile.service.spec.ts]: @meisekimiu done

**Review by meisekimiu — APPROVED** [review]

**Review by k8lyn6 — APPROVED** [review]
I think this looks good and solves the problem of never getting to 100%. There are some further small improvements I think we could make, but I'll make separate tickets for those. Thanks Andrei!

**Pull request merged**